### PR TITLE
Remove `root_name` class method from Serializer, as it is used nowhere.

### PR DESCRIPTION
**Pull request by beauby**

### CODE REVIEW

File: lib/active_model/serializer.rb [hunk 1]
```diff
@@ -98,10 +98,6 @@ def self.adapter
       ActiveModel::Serializer::Adapter.lookup(config.adapter)
     end
 
-    def self.root_name
-      name.demodulize.underscore.sub(/_serializer$/, '') if name
-    end
-
```
Comment (bf4) [inline lib/active_model/serializer.rb]: plz add a changelog entry that this is a breaking change (would be good to see where it was last used?), and I'll merge it

good catch

Comment (beauby) [inline lib/active_model/serializer.rb]: Done

**Comment by beauby** [comment]
Introduced [here](https://github.com/rails-api/active_model_serializers/pull/680/files#diff-5624f9cf35be4dc254412d979fd293c4R98).

**Comment by NullVoxPopuli** [comment]
I'm good with this! nice find!

**Comment by NullVoxPopuli** [comment]
@bf4 should the changelog specifically state that this change is breaking?

**Comment by beauby** [comment]
@NullVoxPopuli As it was part of the public API, we have to inform people it is not anymore.

**Comment by NullVoxPopuli** [comment]
yeah, I re-looked at the changelog and noticed I didn't notice earlier that the changelog entry was under `Breaking Changes`

we're good!

File: lib/active_model/serializer.rb (hunk 1 above)
Comment (joaomdmoura) [inline lib/active_model/serializer.rb]: :+1:

**Pull request merged**